libs/ui/test/test_ui_strings.tsx

```diff
@@ -55,6 +55,7 @@ export const testUiStrings = {
 // Using `testUiStrings.numPlanets()` directly in Storybook, doesn't work, for
 // some reason. This is a convenience wrapper to avoid the failure, which seems
 // related to rendering the count param object as a React component child.
+// istanbul ignore next
```

Comment:
For my understanding, what is this doing?

Reply:
Ah eyah, [`istanbul`](https://istanbul.js.org/) is what we use for instrumenting our source code for test coverage reporting (it comes in as a dependency with `jest`). It reads these comment directives and skips code lines or blocks based on `ignore <type>` - in this particular case, this function is now only used in `libs/ui/src/ui_strings/ui_string.stories.tsx`, so it's not a tested path anymore

Reply:
Ah gotcha, thank you!